src/components/Holdings/HoldingModal.tsx

```diff
@@ -66,7 +75,7 @@ interface ModalProps {
   onRequestClose?: Function
 }
 
-const HoldingModal = (props: ModalProps) => {
+const HoldingModal = (props) => {
```

Comment:
Can we keep the props interface definition? Trying to completely rewrite everything in TypeScript.

Reply:
Added it back in, sorry forgot I had taken it out when I was testing some methods to programatically redirect. Thanks for catching that.

Reply:
Thanks @clarencejychan!